install/build.py

```diff
@@ -226,10 +226,7 @@ def check_cuda_version(compiler, settings):
 
 
 def _format_cuda_version(version):
-    major = version // 1000
```

Comment:
Why do you change this code?

Reply:
Surprisingly version returned from CUDA 9.1 (`libcudart.so.9.1`) is `9010` instead of `9100`.